### PR TITLE
[M1127] attribute/region validations

### PR DESCRIPTION
**Pull request by gridcell**

### CODE REVIEW

**Comment by gridcell** [comment]
[non-region attribute validations](https://trello.com/c/EYLu6e8t/1127-non-region-attribute-validations)

**Comment by gridcell** [comment]
[non-region attribute validations](https://trello.com/c/EYLu6e8t/1127-non-region-attribute-validations)

File: src/api/models/mermaid.py [hunk 1]
```diff
@@ -1007,6 +1007,17 @@ class FishAttribute(BaseAttributeModel):
     class Meta:
         db_table = 'fish_attribute'
 
+    def _get_subclass(self):
```
Comment (saanobhaai) [inline src/api/models/mermaid.py:1010]: Isn't this a duplicate of `_get_taxon`?

Comment (gridcell) [reply src/api/models/mermaid.py:1010]: overlooked

**Pull request merged**